docs/book/features/volume_expansion.md

```diff
@@ -1,120 +1,97 @@
 # vSphere CSI Driver - Offline Volume Expansion
 
-CSI Volume Expansion was introduced as an alpha feature in Kubernetes 1.14 and it was promoted to beta in Kubernetes 1.16. The Vanilla vSphere CSI driver supports offline volume expansion which allows a volume to be extended when it is not attached to a node. Currently the support is available only for block volumes.
+CSI Volume Expansion was introduced as an alpha feature in Kubernetes 1.14 and it was promoted to beta in Kubernetes 1.16. The vSphere CSI driver currently extends this support for offline block volumes only i.e allows a block volume to be extended when it is not attached to a node. Check the [supported features](../supported_features_matrix.md) section to verify if your environment conforms to all the required versions. Note that offline volume expansion is available from vSphere CSI v2.0 onwards in Vanilla Kubernetes and v2.1 onwards in Tanzu Kubernetes Grid (TKG) service. 
 
 ## Feature Gate
 
-Expand CSI Volumes feature is promoted to beta in 1.16 so it is enabled by default. For Kubernetes release before 1.16, ExpandCSIVolumes feature gate needs to be enabled for this feature to support volume expansion in CSI drivers.
+Expand CSI Volumes feature was promoted to beta in kubernetes 1.16, therefore it is enabled by default. For Kubernetes releases before 1.16, ExpandCSIVolumes feature gate needs to be enabled for this feature to support volume expansion in CSI drivers.
 
 ## Sidecar Container
 
-An external-resizer sidecar container implements the logic for watching the Kubernetes API for Persistent Volume claim edits and issuing ControllerExpandVolume RPC call against a CSI endpoint and updating PersistentVolume object to reflect new size.
+An external-resizer sidecar container implements the logic of watching the Kubernetes API for Persistent Volume claim edits, issuing the ControllerExpandVolume RPC call against a CSI endpoint and updating the PersistentVolume object to reflect the new size. This container has already been deployed for you as a part of the vsphere-csi-controller pod.
 
-The resize sidecar container will be deployed in the vsphere-csi-controller pod.
+## Requirements
+If your environment adheres to the required kubernetes and vSphere CSI driver versions mentioned above, this feature will work as is in a TKG cluster. Proceed to the `Expand PVC Example` section below to use this feature.
```

Comment:
`this feature will work as is in a TKG cluster` => `this feature is supported in TKG clusters`

Reply:
The intention was to say that the feature works in TKG without any further changes required to be done. I have already mentioned above that is supported for TKG.

Reply:
`this feature is enabled by default in TKG clusters`?